modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -701,6 +722,20 @@ Test_Region(void)
     DeleteObject(hRgn);
 }
 
+void Test_CursorIcon()
+{
+    BITMAP bmp;
+    HBITMAP hbmMask = CreateBitmap(32, 64, 1, 1, (VOID*)src_mask);
+    /*
+         This test is valid until Win2003Srv sp2.
```

Comment:
What happens on Vista+ systems?

Reply:
Windows7 report 32x64 bitmap. In Vista i dont know

Reply:
Can you check whether this depends on the current Windows' display settings? (256 colors, 16-bit color or higher)

Reply:
![RosTest](https://user-images.githubusercontent.com/49964946/156906462-a2413871-4a14-48ee-bbbd-f99e5df29dc2.png)
Here it shows 4 tests in the following order:
800x600x256, 1024x768x16bits, 1600x1200x32bits and 800x600x24bits

Reply:
I was asking testing that on Windows 7, not ReactOS.

Reply:
![Win7Test](https://user-images.githubusercontent.com/49964946/156918182-a04dbc29-8bbb-4b57-9940-845c64053bb0.jpg)
16 and 32 bits from 800x600, 1280x600 and 1366x768 resolution